src/commands/run-script.js

```diff
@@ -1,14 +1,14 @@
-#!/usr/bin/env node
-
 /* eslint-disable no-console */
 
 const assert = require('assert');
 const fs = require('mz/fs');
 const path = require('path');
 const git = require('@financial-times/git');
-const rmfr = require('rmfr');
 
-const { workspacePath } = require('../lib/constants');
+const { GitProcess } = require('dugite');
+const constructDugiteExecArgs = require('@financial-times/git/src/helpers/construct-dugite-exec-args');
```

Comment:
`@financial-times/git` doesn't exist anymore right?

Reply:
[?????](https://github.com/Financial-Times/tooling-helpers/tree/master/packages/git)

Reply:
Sorry, you are right, I was checking https://github.com/Financial-Times/git, forgot it was in https://github.com/Financial-Times/tooling-helpers/tree/master/packages/git